.circleci/ci-setup.sh

```diff
@@ -3,14 +3,28 @@ set -e
 
 # intended to be run from directory above this one.
 
-# Setup a proper path, I call my virtualenv dir ".venv"
-PATH=$WORKSPACE/.venv/bin:$PATH
+# Setup a proper path, I call my virtualenv dir ".venv_non_dev"
+PATH=$WORKSPACE/.venv_non_dev/bin:$PATH
 python3 --version
-if [ ! -d ".venv" ]; then
+if [ ! -d ".venv_non_dev" ]; then
         # use python3 to create .venv
-        python3 -m venv .venv
+        python3 -m venv .venv_non_dev
 fi
-source .venv/bin/activate
+source .venv_non_dev/bin/activate
```

Comment:
So sorry you ran into this issue. Makes me think we really need to investigate moving to `poetry` or something that does a nicer job of differentiating runtime vs development dependency sets.

Eventually, we'll probably have to jump through similar flaming hoops for the internal build (see: Jenkinsfile) when a non-dev environment is needed during the CI build to self-scan.